src/homekit/accessories/HttpWebHookSensorAccessory.js

```diff
@@ -163,4 +151,27 @@ HttpWebHookSensorAccessory.prototype.getServices = function() {
   return [ this.service, this.informationService ];
 };
 
+HttpWebHookSensorAccessory.prototype.resetToInitialState = function () {
+    if (this.type === "contact") {
+        this.storage.setItemSync("http-webhook-" + this.id, true);
+        this.service.getCharacteristic(Characteristic.ContactSensorState).updateValue(Characteristic.ContactSensorState.CONTACT_DETECTED, undefined, Constants.CONTEXT_FROM_TIMEOUTCALL);
+    } else if (this.type === "motion") {
+        this.storage.setItemSync("http-webhook-" + this.id, false);
+        this.service.getCharacteristic(Characteristic.MotionDetected).updateValue(false, undefined, Constants.CONTEXT_FROM_TIMEOUTCALL);
+    } else if (this.type === "occupancy") {
+        this.storage.setItemSync("http-webhook-" + this.id, false);
+        this.service.getCharacteristic(Characteristic.OccupancyDetected).updateValue(Characteristic.OccupancyDetected.OCCUPANCY_NOT_DETECTED, undefined, Constants.CONTEXT_FROM_TIMEOUTCALL);
+    }
+}
+
+HttpWebHookSensorAccessory.prototype.setAutoReleaseTimeout = function () {
+    if (!this.autoRelease || !this.resetAutoReleaseOnDetection) return;
+    clearTimeout(this[this.getTimeoutKey()]);
```

Comment:
Clearing and resetting timeout should only be done if `resetAutoReleaseOnDetection` is true. Otherwise it should always add the timeout as before, so nothing break.